wolfcrypt/src/asn.c

```diff
@@ -28314,7 +28314,12 @@ int wc_EncodeNameCanonical(EncodedName* name, const char* nameStr,
 int ParseKeyUsageStr(const char* value, word16* keyUsage, void* heap)
 {
     int ret = 0;
-    char *token, *str, *ptr;
+#ifdef WOLFSSL_NO_MALLOC
+    char str[1024];
```

Comment:
Not a big fan of this hardcoded "1024". Can we at least make a define like MAX_KEY_USAGE_STRING

Reply:
`asn.c` changes wound up being unneeded -- reverted.